docker/packager/binary/Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN apt-get update \
         /etc/apt/sources.list \
     && apt-get clean
 
+# FIXME: clang-13 is here for broken TSAN
```

Comment:
The difference will be between:

"oh, they write a crap instead of code"

and 

"ClickHouse is one of the most well-tested codebases in C++, and we found bugs in LLVM-14 ThreadSanitizer by Google, and we even use this fact for promotion".

Reply:
You said

> Not for changelog (changelog entry is not required)

but we should actively advertise this achievement on conferences and everywhere else, including the changelog.

Reply:
Sure, but I can't find a related task. https://github.com/ClickHouse/ClickHouse/issues/38019 looks fixed by https://github.com/google/sanitizers/issues/1540. Dmitry doesn't aware of the "tsar-ticket" for TSAN issue. Maybe @tavplubix has something on top of the head?

Reply:
Yep, https://github.com/google/sanitizers/issues/1540 is fixed, but AFAIK the fix is not backported to clang-14. And the ticket for performance degradation is https://github.com/google/sanitizers/issues/1552. Btw, seems like it's almost fixed too, so maybe it makes sense to wait a bit and upgrade to clang-15 (trunk)... Or downgrade to clang-13 for now and try clang-15 when build with the fix will be available in llvm repo.

Reply:
Good, I changed the PR category and added links to tickets. Now I'll add a revert commit for https://github.com/ClickHouse/ClickHouse/pull/38106

Reply:
see https://github.com/ClickHouse/ClickHouse/commit/78c4d4f92a2521ddfcb5f6d22398f8ed96cbdcd2